app/routes/quotes/components/Quote.tsx

```diff
@@ -93,17 +94,15 @@ const Quote = ({
                   iconName="ellipsis-horizontal"
                 >
                   <Dropdown.List>
-                    <Dropdown.ListItem>
-                      <button
-                        onClick={() =>
-                          quote.approved
-                            ? dispatch(unapprove(quote.id))
-                            : dispatch(approve(quote.id))
-                        }
-                      >
-                        {quote.approved ? 'Fjern godkjenning' : 'Godkjenn'}
-                      </button>
-                    </Dropdown.ListItem>
+                    {!quote.approved && pippi ? (
+                      <Dropdown.ListItem>
+                        <button onClick={() => dispatch(approve(quote.id))}>
+                          {'Godkjenn'}
+                        </button>
+                      </Dropdown.ListItem>
+                    ) : (
+                      <div></div>
```

Comment:
Instead of using a ternary, just use && so that we don't need empty divs like this in the dom